addons/calendar/models/calendar_event.py

```diff
@@ -930,7 +930,8 @@ def _split_recurrence(self, time_values):
             # When we try to change recurrence values of an event not following the recurrence, we get the parameters from
             # the base_event
             previous_week_day_field = weekday_to_field(self.recurrence_id.base_event_id._get_start_date().weekday())
-        self.write(time_values)
+        # Send context key 'send_updates' as False in edit option 'This and following events'
+        self.with_context(send_updates=False).write(time_values)
```

Comment:
in calendar:
```py
def _get_split_context():
    return {}
``` 
In gogle_calendar:
```py
def _get_split_context():
    context = super()._get_split_context()
    return context.update(send_updates=False)
```

Reply:
I tried using {'dont_notify': True} but no success: it was sending an outdated 'Updated invitation' email, so I kept the {'send_updates': False} context key

Reply:
but you may use both context key:
one is used to handle internal notifications (dont_notify) and the other is used by google: send_updates.

Reply:
Good idea! I will try it

Reply:
I'm trying to solve a problem: sometimes I receive "None" from the self._get_split_context() call, and it throws an error with the following message:
```
TypeError: argument of type 'NoneType' is not iterable

The above server error caused the following client error:
null
```